conf/rh_cloud.yaml.template

```diff
@@ -0,0 +1,2 @@
+RH_CLOUD:
```

Comment:
Please add a basic validator for this file. I'm thinking TOKEN itself shouldn't be commented out, but should just have an empty value.

https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/validators.py

Reply:
updated the pr.